source/getting_started/aiot_intelligent_edge/coco_for_applications/analytics_client_applications/browser_js_analytics_client_applications/sample_app_walkthrough_browser_js_analytics_client_apps.rst

```diff
@@ -3,9 +3,128 @@
 Sample App Walkthrough
 ======================
 
-.. figure:: /_static/coming-soon.png
-   :align: center
+.. sectionauthor:: Deekshith
 
-   **Coming Soon**
+Connect to COCO
```

Comment:
Please see example of code walkthrough formatting/style in COCO Device SDK Quick Start Guide

Reply:
Handled followed the format used by akshay and krishna